infra/charts/feast-feature-server/templates/otel-collector.yml

```diff
@@ -0,0 +1,56 @@
+# API reference https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md
+# Refs for v1beta1 config: https://github.com/open-telemetry/opentelemetry-operator/issues/3011#issuecomment-2154118998
+{{- if .Values.metrics }}
+apiVersion: opentelemetry.io/v1beta1
+kind: OpenTelemetryCollector
+metadata:
+  name: otelcol
+  namespace: {{ .Release.Namespace }}
```

Comment:
this line shouldn't be necessary ... helm should deploy to the appropriate namespace. the existing templates don't specify this line, so we should probably follow suit